### ISSUE TITLE
Passwords are not loaded in Android

### ISSUE THREAD

**Issue by mhabibi** [SpiderOak/Encryptr#292]
Since a few days ago I have not been able to fetch passwords on my Android devices. I checked in two Androids, both Android 6.
It gets stuck in "DECRYPTING ENTITIES..." and never shows anything else.

**Comment by devgeeks** [comment]
Can you tell me what version of Android you have tried?

I cannot reproduce on the Android device I have with me at the moment.

**Comment by exterm** [comment]
I have a moto g4 plus with Android 7.0 . I purged the data, uninstalled the app, both did not help.

**Comment by unclescott** [comment]
mhabibi, thank you for posting,
I see this on Android 7.0, security patch level March 5, 2017, on a Nexus 6 phone.
Encryptr continues to run OK for me on an iPhone and my laptop.

**Comment by devgeeks** [comment]
Thanks

**Comment by droidbag1** [comment]
Hi, I am having same issue. Using phone moto x2nd gen, android 5.1 android security patch 8/1/16. Just says decrypting entries after logging in. Let it run for 15 min with no result. Tried restart and uninstall. Have not gotten any security patches or any changes recently. Started happening 2-3 days ago. Please resolve.

**Comment by MrWiggle** [comment]
Hi everyone, I'm having exactly the same issue - and it is so frustrating!  I've been using Encryptr on my Mac and my android HTC phone for about the last year, and it's worked great - until this weekend!!

It's still working fine on my mac, but has stopped working on my Android.  I can log in about 50% of the time but, even when I can log-in, it seems to hang at "Decrypting Entries".  Weirdly, when I let it run my phone gets super hot, and runs through it's battery in record time, so it seems like its doing something, just not anything I can see, and I can't get into my passwords.

I'm running Android 5.0.2 on an HTC One m7.  It's old, I know, but it's worked great until now!!  I've tried uninstalling and reinstalling it, and there have been no recent updates and/or security patches. 

All advice VERY gratefully received!!!  I want to keep using Encryptr, and really, REALLY don't want to have to re-enter all my passwords into some new programme!!!

**Comment by jfvrxmbl** [comment]
Hi,
I have the same issue on SONY Xpéria Z3 running on Android 6.0.1 without particular causes.
Start happening a week ago
Thanks for your help.

**Comment by rhinosau** [comment]
I'm having the same issue on LG Nexus 5 with Android 6.0.1.

**Comment by devgeeks** [comment]
The breadth of devices this is occurring on is disturbing.

I am flummoxed that I cannot replicate it. 

Paging @reneelynn and the rest of the SpiderOak team

**Comment by citizenspress** [comment]
Same issue on a OnePlus One after upgrade to LineageOS 14.1 (Android 7.1.2) from Android 7.0. Has not been working since update to 7.1.

Have tried wiping all files associated with Encryptr and re-installing the application with no effect.

**Comment by devgeeks** [comment]
Quick question for everyone:

Roughly how many entries do you have stored in your account? I have < 70 and it still took quite a while to sync. I have to wonder if something has slowed the server down or something causing a timeout?

**Comment by mhabibi** [comment]
I have about 85 passwords and other entities in total.

**Comment by MrWiggle** [comment]
I have about 50 passwords/other entries.

**Comment by droidbag1** [comment]
Anyone working on this? Anyone find a good alternative - free is better, with backups to a cloud is ideal. Likely to splurge $10 on enpass, if this does not start working soon.

**Comment by MrWiggle** [comment]
Hi everyone,

I have a partial, if frustrating, update.  I did a complete "factory re-set" on my HTC, and re-installed everything. 

Unfortunately, now Encryptr is no longer letting me log in at all.  I enter my credentials, it appears to be working, and then just rejects my log-in.  Of course, I've checked my log-in credentials multiple times - and they're the same credentials I use for Encryptr on my other devices.

So, I'm now totally out of ideas, and incredibly frustrated.  I have to leave for a business trip tomorrow AM and I don't have access to my passwords, etc on the move. 

In a way, it's kind of a compliment to Encryptr, I hadn't realised how much I had begun to rely on it - but, with it not there, I am really stymied!!!!  I have a feeling I'm going to have the find an alternative soon, and start to re-enter all those passwords....groan!!!!

**Comment by EarthLord** [comment]
Same here. I've about 50 entries. Gets stuck at the decrypting entries stage. On a Nexus 5 with a custom ROM having Android 7.1.2.

**Comment by tonym07** [comment]
nexus 6 running Android 7.1 Stock with about 86 passwords .
When i open the Application it says on top of the screen 
Local Cache invalid Loading from server and it just keeps saying after that decrypting  entries and nothing shows

Hopefully this will help in a way

**Comment by MrWiggle** [comment]
Hi everyone,

I'm very sad to say that, after trying everything I could think of, and tearing my hair out for the last week, I've now migrated to EnPass (ie, just downloaded EnPass, and re-entered everything manually).  It's working well on both my Mac and Android devices, and meets my needs for usability and security - with the added bonus that I now have "off line" access, which may prove useful.

I have to admit, it's not nearly as elegant as the Encryptr interface, which I think is really a masterpiece of simplicity, and so I don't enjoy using it nearly as much.  But, I just can't do without a password manager - without it, I was really struggling.  

Good luck - I hope someone is able to get Encryptr back on track!

**Comment by franchb** [comment]
I'm very sad too. Last 1,5 years I've been using Encryptr. Today I've migrated to EnPass.

**Comment by MrWiggle** [comment]
Hi frenchb, I feel your pain - EnPass works, but it's not nearly as elegant or easy as Encryptr was....

Encryptr (when it worked), made my life so much easier - now I've got EnPass, and so a million crappy labels, folders and windows to worry about...sigh....

**Comment by deepdivedylan** [comment]
I have the same problem with a Nexus 6 and Samsung Galaxy Tab S2. I have > 150 passwords and love Encryptr and don't want to migrate. 😿

**Comment by sudomake** [comment]
I was using Encryptr without issue for a long time.  One day I received the Local Cache Invalid message during a sync.  After that I could never get past decrypting.  Uninstalling/Clearing the application data did not help.

I did a factory reset which did fix the problem until I received the Local Cache message again.  Now back to the same problem.  Works great on OSX, iOS, and linux though!

Samsung Galaxy S4
Android 5.0.1
Android Security Patch 2015-11-01

**Comment by DrShittler** [comment]
I just migrated to Padlock, which is still open source (as Encryptr used to be) and comes very close to the elegance of our beloved Encryptr.
I'll be using both for some time to give SpiderOak the chance to fix things but I'm afraid they won't - just as the program hasn't received an update in quite some time. I sense that SpiderOak wants to get rid of Encryptr as it is not profitable at all and due to the highly sensitive data they are storing a big PR risk - so it still needs a lot of attention encryption-wise. The "issue" on android devices could be a first step towards losing users on purpose.

**Comment by A-Stevens** [comment]
Same Issue, Kyocera DuraForce XD. Tried re-install, factory reset and re-install, and fresh install on new device (replacement for unrelated hardware issue). and problem persists.
Encryptr syncing without issue on two PCs and Asus TF300t tablet.

**Comment by student237** [comment]
I have the same problem with my Android installation of Encryptr (my Linux desktop version of the application still works great). I have 65 passwords and entities stored that I can't access via mobile platform as the app hangs on "DECRYPTING ENTRIES".  I've tried reinstalling the app, renaming the data folder, etc., with no resolution to the problem. I reached out to SpiderOak support three weeks ago and received the same canned response (almost verbatim) as @unclescott received. SpiderOak knows they have a problem, but they don't appear to be very interested in finding a fix. Perhaps @DrShittler is onto something with the statement: "The "issue" on android devices could be a first step towards losing users on purpose."   It's really too bad, as I love the security and simplicity of the product. I guess it is time to start investigating alternatives. 

Samsung Note4 (SM-N910R4)
Android 6.0.1
Android Security Patch 2017-03-01

**Comment by dodys** [comment]
Same issue on a:
OnePlus X
Android 6.0.1
Android Security Patch 2016-11-01

**Comment by TheLioness22** [comment]
Some issue on Motorola Droid Turbo.  Android 6.0.1.  Security patch: January 1, 2017.   Tried force stopping the app, clearing cache, clearing data, uninstalling and then reinstalling.  It opens and lets me enter credentials, then says "Decrypting Entries..." and never gets past that screen.  It was working until a few weeks ago -- likely around the time that this thread was started.  I initially assumed it was some kind of back-end connection error and that time would resolve it.  However, it is still not working.  Given the vast number of people on this thread who are experiencing issues, it is clear that something has gone awry in the app -- most likely in regards to either syncing the data or decrypting it.  The desktop app is still working, so it is clear that the data does still exist and can be decrypted (so data corruption or removal can be ruled out).  A fix would be greatly appreciated.  If I knew anything about mobile app development, I'd dig into the code on here, but that is well outside of my expertise.

**Comment by Halfnium** [comment]
I see exactly what others are reporting, this time on a Samsung Galaxy Amp Prime 2 'phone running Android 7.0.  By "exactly," I mean:

1) Encryptr worked well when I first installed it on the 'phone, but subsequently seemed to hang either upon login or upon sync and decryption of items previously stored on the server.

Oddly, the program would hang in different places from one trial to another.  This suggests something timing-dependent.

2) The 'phone's folder named org.devgeeks.encryptr is indeed present and contains the subfolders that TheLioness22 describes.  Both of these subfolders are empty, as TheLioness22 further describes.

3) Encryptr still works fine on my Ubuntu 16.04 (LTS) and Windows 10 systems.

This is after many careful install / shut down / startup / run experiments on the 'phone.

I now stand embarrassed that I recently recommended Encryptr to an online audience of older travelers, people who greatly need a simple, robust, cross-platform password manager.  Old folks tend not to protect their 'phones with login credentials, making password management a critical issue for them.

The joke's on me.  I've used the SpiderOak file sharing service for years with no problems whatsoever.

**Comment by exterm** [comment]
Yeah, I am also a paying SpiderOak backup user. I would pay a few dollars a month for encryptr even. This current complete outage on android for over a month is really bad. Is Encryptr even still officially supported by SpiderOak?

**Comment by mhabibi** [comment]
Today I am getting an error on Mac, this happens when I remove an entity. I post it here, although I think it's a different bug but maybe it helps.
<img width="963" alt="screen shot 2017-08-03 at 17 02 31" src="https://user-images.githubusercontent.com/4964205/28930747-360e0482-7874-11e7-8eb6-653c6c38246d.png">

**Comment by devgeeks** [comment]
I hate to ask, but has anyone in this thread contacted SpiderOak support?

It might just be that they are not watching the GitHub issues.

https://support.spideroak.com

**Comment by dodys** [comment]
Yes, I have and got the same reply as comment 4.

**Comment by Samahedron** [comment]
Kelly Swezey (SpiderOak Support) Aug 2, 1:57 PM CDT

Hey [Samahedron]!

My  name is Kelly with SpiderOak and I will be happy to assist you. It's  currently being worked on, but I cannot provide an ETA. I wish I could,  but due to the complex QA sprints we perform, I'm unable to correctly  guess or predict when it will be resolved.

I appreciate your time and I'm glad I could respond in a timely manner!

Take care,
Kelly

**Comment by student237** [comment]
Based on @Samahedron 's recent support response, it looks like SpiderOak is attempting to correct the issue as opposed to scrapping the project. That's excellent news! The email I received a month ago wasn't nearly as encouraging, but I've been holding out on switching applications, checking in on the thread every few days, hoping that SpiderOak would finally step up and help us out. 
_______________________________________________________

Warren Post (SpiderOak Support) Jul 5, 2:04 PM CDT

Hi [student237], this is Warren from SpiderOak. That does sound frustrating and I regret this problem. Worse, I'm afraid that there's very little help that we are able to provide. When we designed Encryptr, we made design decisions that maximize privacy rather than enable troubleshooting, so we don't have access to any client or server logs, or any data at all that isn't encrypted with keys that we don't have. I'm afraid that all I can suggest is the following two trial and error approaches, neither of which are of help to you:

    Try on a different device. However that does not address your problem of accessing Encryptr on your phone specifically.
    Try renaming the application data folder. However you've already done that.

Sorry for the bad news.

Apropos that, this is why we are considering discontinuing Encryptr. Its design makes it impossible for us to properly support and we don't like leaving people like you in the air like this. That said, if we were to do this it would be only after adding a data export feature and providing prior notice, so no one would be left without a way to extract their data.

**Comment by aleixx** [comment]
Same issue with:

_**Nexus 5
6.0.1**_

**Comment by Samahedron** [comment]
I followed up on the Android issue with Spideroak support. Their response below:

I just put this question to Spideroak customer service. Their response below:
--------
 Kat Tyler (SpiderOak Support)

Sep 3, 1:32 PM CDT

Hi [Samahedron],

I'm Kat with SpiderOak Support. Thanks for contacting us!

Our developers are currently working on a new version of Encryptr, which should address the issues some Android users have been seeing when trying to load the application. I'm unfortunately unaware of an ETA on when that might be released - we try not to release new versions until they're completely ready and have been thoroughly tested.

That new version will be the last release for this iteration of the product, as we're end-of-lifing Encryptr as it exists now. That new version will include an export feature, but won't receive further updates or added features. Encryptr may come back in a new form at some point in the future, but I don't have concrete details on if or when that might happen.

I wish I had better news for you, or could even give you a timeframe to work with, but I've shared with you everything I know about the current state of Encryptr. If there's anything more we can do, please let me know.

Thanks,
Kat

**Comment by julian-higginson** [comment]
same problem, google pixel. 

Really disappointed in this as it caused a massive inconvenience to me yesterday then i discover it's been a problem for some users for some for months, and I had no warning of this... you guys should push out a warning to every android user of the app at a minimum (or an EOL warning to everyone!) so people don't get caught out...

It would have been better if you'd made this a paid service, then used the money to develop it better, rather than just drop it and leave people hanging with a bunch of passwords they can't get to! 

This kind of silent failure makes me worried for spideroak as a product.

**Comment by lowellday** [comment]
If Encryptr has reached end-of-life **PLEASE** make a formal announcement and provide users with a way to export their data!

**Comment by convct** [comment]
Moto G5+ Android 7.0
Android security patch level 1st Jan 2017
Kernel Feb 6 20108 2018
Build number NPN25.137.33
Stopped syncing around 2 weeks ago I think. I have not yet checked updates installed.
if there is anything I can do to help test this please let me know.

**Comment by JackDraak** [comment]
No way to pin this to the top, so, here it is again, quoted from above, for those who might find it most readily so:

Kat Tyler (SpiderOak Support)

Sep 3, 1:32 PM CDT

Hi [Samahedron],

I'm Kat with SpiderOak Support. Thanks for contacting us!

Our developers are currently working on a new version of Encryptr, which should address the issues some Android users have been seeing when trying to load the application. I'm unfortunately unaware of an ETA on when that might be released - we try not to release new versions until they're completely ready and have been thoroughly tested.

That new version will be the last release for this iteration of the product, as we're end-of-lifing Encryptr as it exists now. That new version will include an export feature, but won't receive further updates or added features. Encryptr may come back in a new form at some point in the future, but I don't have concrete details on if or when that might happen.

I wish I had better news for you, or could even give you a timeframe to work with, but I've shared with you everything I know about the current state of Encryptr. If there's anything more we can do, please let me know.

Thanks,
Kat

**Comment by RatherFunky** [comment]
hmm I might have found a temporary fix until update comes out. Here is the answer: if you are go pass the login screen and can see + sign press it. Then create random entry. Then save it. All my passwords loaded back. I hope it will work for you all. The entry which I tested was password.

**Comment by Samahedron** [comment]
cross post from:https://github.com/SpiderOak/Encryptr/issues/296


> SpiderOak messaged me on Twitter that should be added to this thread.TL; DR: Encryptr is being moved to the Flow platform, and not going EOL as previously posted on these threads.https://twitter.com/deepdivedylan/status/918210143606669315It's been a while since an update, but they are coming! A complete rewrite will be out early next year too.That work has already started and is progressing nicely, but we’re nearing an intermediate release with export and offline mode.We are migrating Encryptr to the Flow platform (which Semaphor is built on) and will continue developing it as a SpiderOak product.Right, we need to put a formal announcement there. For a time the plan was to EoL it, but that’s not true now.
--

**Comment by JulianMeans** [comment]
well, it sure looks like "early 2018" for a rewrite of the application has blown out a bit. it's now early 2019.... 

Funnily enough, my encryptr started working on my phone (the same phone it had STOPPED working on a year and a half ago) but there's no new app, still. The latest release of encryptr on google play is still dated 2015...
